backend/middleware/errorHandler.js

```diff
@@ -2,12 +2,17 @@ const { serviceUnavailableMessage } = require('../common-messages')
 const { logError } = require('../logError')
 
 const errorHandler = (error, req, res, next) => {
+  if (error.response.status === 404) {
+    res.status(404)
+    return res.render('notFound.njk')
```

Comment:
think this should also include the `originalUrl` in the render so that `Continue` button works.  Might be better to do this in a separate PR as should really have an int test for it too

Reply:
yes - will pull new error handling to separate PR